contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -29,6 +29,7 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
 
   address[] internal s_keeperList;
   EnumerableSet.UintSet internal s_upkeepIDs;
+  EnumerableSet.UintSet internal s_pausedUpkeepIDs;
```

Comment:
definitely want to avoid this if we can. I don't really see what value it adds. I think we can get away with just adding a boolean to the Upkeep struct

Reply:
this is mainly for get active upkeep ids.
if we only add a boolean to upkeep struct, it makes `getActiveUpkeepIDs` difficult bc we will not know how many upkeeps are paused before the `startIndex`